src/Common/HashTable/Hash.h

```diff
@@ -48,6 +48,49 @@ inline DB::UInt64 intHash64(DB::UInt64 x)
 #include <arm_acle.h>
 #endif
 
+#if (defined(__PPC64__) || defined(__powerpc64__)) && __BYTE_ORDER__ == __ORDER_LITTLE_ENDIAN__
+//CRC32 Implementation for POWER
+inline uint32_t ppc_crc32_u8(uint32_t crc, uint8_t v)
+{
+    uint32_t _crc = crc;
+    _crc ^= v;
+    for (int bit = 0 ; bit < 8 ; bit++)
+    {
+       if (_crc & 1)
+          _crc = (_crc >> 1) ^ UINT32_C(0x82f63b78);
```

Comment:
This code looks like a copy-paste
(because you have written `UINT32_C(0x82f63b78)` and `int bit` and `bit++` instead of `++bit` and extra whitespace and `_crc`, etc)
but you did not specify the source.

Reply:
But now the hash function will be extremely slow... most likely this is not what we need.

Reply:
@alexey-milovidov , This function I basically doing the same what intel intrinsics does, 
below are the sources to get the algorithm for intel.

https://masterchef2209.wordpress.com/2020/06/17/guide-to-intel-sse4-2-crc-intrinisics-implementation-for-simde/

https://www.intel.com/content/www/us/en/docs/intrinsics-guide/index.html#text=_mm_crc32_u64&ig_expand=1497

Yes,  we also observed the degrade in performance for weakhash calculation for POWER.